src/yogeshNagar/A15_ArrayLogic.java

```diff
@@ -0,0 +1,88 @@
+package yogeshNagar;
+
+public class A15_ArrayLogic {
+	
+	int getMaxArray(int[] arr) {
+		int maxArray = 0;
+		for(int index=0;index<arr.length;index++) {
```

Comment:
If you are taking the 0th index as max number then you can start the loop from index 1 as well, same for other methods